<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tuni Travel</string>
  </property>
  <widget class="QLabel" name="label_signin_pic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_etatlogin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_69">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>41</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_id">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_signin_password">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_signin_login">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineEdit_signin_username">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
